src/server/model/email.js

```diff
@@ -17,7 +17,7 @@ const createSes = () => new aws.SES({ apiVersion: '2010-12-01' })
 
 const emailOpts = (subject, html, location, site) => {
   let emailExtension
-  if (emails.includes(location.toLowerCase())) {
+  if (location && emails.includes(location.toLowerCase())) {
     emailExtension = location.toLocaleLowerCase()
   } else if (emails.includes(site.toLocaleLowerCase())) {
     emailExtension = site.toLowerCase()
```

Comment:
should we do `toLocaleLowerCase` here too? I can't remember why we did that anyway...

Reply:
I don't think we need it here, we are just checking if it's undefined.